MAINTAINING.md

```diff
@@ -0,0 +1,102 @@
+# Maintaining Amundsen
+
+As maintainers of the project, this is our guide. Most of the steps and guidelines
+in the [Contributing](CONTRIBUTING.md) document apply here, including how to set
+up your environment, write code to fit the code style, run tests, craft commits
+and manage branches.
+
+Beyond this, this document provides some details that would
+be too low-level for contributors.
+
+## Table of Contents
+
+- [Communication](#communication)
+- [Managing the community](#managing-the-community)
+- [Workflow](#workflow)
+- [Architecture](#architecture)
+- [Updating the changelog](#updating-the-changelog)
+- [Documentation](#documentation)
+- [Versioning](#versioning)
```

Comment:
should we remove versioning and releaseing for now?

Reply:
Yeah, thanks for calling it out!